supporting_files/AtlasSiteConfig.yml

```diff
@@ -81,7 +81,7 @@ atlas_r_installation: sw/atlasinstall_prod/R_install
 cttv_efo_sparql_script: sw/atlasinstall_prod/atlasprod/db/scripts/get_efo_in_cttv_validation_data.sh
 
 # File containing property types and values to exclude from Zooma mapping.
-zooma_exclusions_file: <CONDA_PREFIX>/atlasprod/supporting_files/zooma_exclusions.yml
+zooma_exclusions_file: <CONDA_PREFIX>/atlasprod/perl-atlas-modules/supporting_files/zooma_exclusions.yml
```

Comment:
Just to confirm, are we moving zooma exclusions away from its current location? So I should get it from this Conda package in future? Agree @pcm32 ?

Reply:
given that Jon is also using it from that location, I would strongly suggest that we avoid the path name changes in favour of a symlink that sorts the issue out, without changing the path in any of our repos.

Reply:
@pcm32 I can very easily adapt to use the file from wherever (file system/ Git repo/ Conda package), so don't worry about that.

Reply:
But I'm not sure putting it in the Conda tree is a good idea (unless it's packaged that way- you can see my confusion here)